evap/contributor/views.py

```diff
@@ -71,9 +71,6 @@ def course_view(request, course_id):
     if not (course.is_user_editor_or_delegate(user) and course.state in ['prepared', 'editor_approved', 'approved', 'in_evaluation', 'evaluated', 'reviewed']):
         raise PermissionDenied
```

Comment:
as both @He3lixxx and I needed longer than necessary to parse this condition, and the erroneous condition you removed probably came from that, could you change this condition to `if not ... or course.state not in ...`?